WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -857,8 +872,8 @@ - (NSPredicate *)predicateForFetchRequest
 {
     NSPredicate *predicate;
 
-    if (self.postIDThatInitiatedBlock) {
-        predicate = [NSPredicate predicateWithFormat:@"topic = %@ AND (isSiteBlocked = NO OR postID = %@)", self.readerTopic, self.postIDThatInitiatedBlock];
+    if ([self.postIDsForUndoBlockCells count]) {
+        predicate = [NSPredicate predicateWithFormat:@"topic = %@ AND (isSiteBlocked = NO OR postID IN %@)", self.readerTopic, self.postIDsForUndoBlockCells];
```

Comment:
I remember when Core Data didn't support IN clauses with array objects. :stabby: